multicast/recv.py

```
@@ -211,7 +212,95 @@
+_w_example_code: str = """
+sock = multicast.genSocket() if isock is None else isock
+sock.bind((multicast._MCAST_DEFAULT_GROUP, port))
+
+"""
+
+
+_w_advice: str = "...".join([
+	"Consider using",
+	_w_example_code,
+	"instead, for improved performance. Otherwise specify the multicast bind group.",
+])
```

Ruff (0.8.2)

Consider f-string instead of string join

Replace with f-string

(FLY002)